Add TDClassifier and ESRGAN Examples

jpyepez opened the pull request:
Added TDClassifier: MobileNet example for Touchdesigner

Commit:
added TDClassifier

Commit:
updated README.md

cvalenzuela commented:
We should remove the [`.DS_Store`](https://github.com/runwayml/touchDesigner/pull/1/files#diff-edb274d8c594429b8ccc684d79d771f6) files in your commit. I created a `.gitignore` in master now.

Also, this example only shows how to load a `.json` file inside TD, no? You are not streaming data to or from the app?

Commit:
removed ds store files

jpyepez commented:
Got it. Just removed the `DS_Store` files.

And about the `json` file, that's correct. I was going to start with OSC, but we had the issues I mentioned in the `#bugs` channel. I figured a couple of examples with `json` and `csv` would be a good spot to start. Already working on a couple of HTTP examples though, which would require streaming.

cvalenzuela commented:
Sounds good. Do you wanna wait for those new examples before merging?
Also, this might be helpful: https://github.com/BarakChamo/TD_PoseNet

jpyepez commented:
Oh nice, that's a cool example! We could just merge this one for now. Working on the P5js one with HTTP first, so that one would go under the other repo.

cvalenzuela commented:
Great, I'll leave this PR open for now then!

Commit:
added EnhanceTextures example

Commit:
updated README with ESRGAN link

Commit:
cleaned up folder

jpyepez commented:
@cvalenzuela -- I have added the ESRGAN example. This one has a bit of a tricky flow, but I don't think it'll be that hard to figure out. Added a couple of notes at the end, one related to the `filename` HTTP attribute that we talked about recently. Let me know if there's anything to add/change.

cvalenzuela commented:
Nice! This is a super useful example. I was trying the demo but when to step 5 (`Open the EnhanceTextures.toe file and press 'F1' to enter performance mode.`) I get the following screen:
![image](https://user-images.githubusercontent.com/10605821/56173415-6cc6c600-5fbb-11e9-9a30-a92c7c575fa7.png)

I haven't used TD much, so perhaps I might be missing something!

jpyepez commented:
Got it! Managed to repro using the TD 2018 build (latest official build) and I found the issue. I'll hit you up on slack to explain, since it raises some interesting questions you all might want to consider for Runway-Touchdesigner integration in general.

Commit:
modified for TD2018 build

Commit:
updated README

jpyepez commented:
All done! Should be compatible with the current build (2018) now. Added a few notes to the README.

cvalenzuela commented:
Works great! Thanks!
